automation/features/pageobjects/governance.page.js

```diff
@@ -1,18 +1,18 @@
 import Page from "./page";
 
 class GovernancePage extends Page {
-  get proposalStatesTestId() {
-    return "governance-proposal-state";
-  }
-  get proposalTableTestId() {
-    return "governance-proposal-table";
-  }
-  get propsalClosingDateTestId() {
-    return "governance-proposal-closingDate";
-  }
-  get proposalEnactmentDateTestId() {
-    return "governance-proposal-enactmentDate";
-  }
+    get proposalStatesTestId() {
+        return "governance-proposal-state";
+    }
+    get proposalTableTestId() {
+        return "governance-proposal-table";
+    }
+    get propsalClosingDateTestId() {
+        return "governance-proposal-closingDate";
+    }
+    get proposalEnactmentDateTestId() {
+        return "governance-proposal-enactmentDate";
+    }
 }
 
 module.exports = new GovernancePage();
```

Comment:
chore (blocking): We should export the classes and create instances within all of the tests to ensure tests are deterministic and state is not leaked in between tests

Reply:
we can have a ticket for this to look into it but it's not part of this work

Reply:
Which ticket?

Reply:
https://github.com/vegaprotocol/token-frontend/issues/1282